test/test.js

```diff
@@ -25,6 +26,21 @@ tape("hash is the same as node's crypto", function (t) {
   t.end()
 })
 
+tape("hash is the same as node's crypto for all algos provided by node", function (t) {
+  var hashes = crypto.getHashes()
+  Object.keys(sha).forEach(function (alg) {
+    if (hashes.indexOf(alg) === -1) return // skip unsupported by current Node.js version
```

Comment:
Do we really test newly added `sha512-224` / `sha512-256`?

Reply:
@fanatid https://github.com/crypto-browserify/sha.js/pull/73 should fail.

But it looks like CI is disabled for some reason?